poradnia/cases/models.py

```diff
@@ -51,12 +51,10 @@ def by_involved_in(self, user, by_user=True, by_group=False):
 
     def by_msg(self, message):
         cond = Q()
```

Comment:
Zbędne już. Aby budować zapytania alternatywne w Django należy użyć obiektu Q(). Można wtedy zrobić zapytanie:
```
Case.objects.filter(Q(pk='5') | Q(pk='6') | Q(created_by__username='5')).all()
```
przy czym ``Q(5) | Q(5)`` daje nowy obiekt ``Q``.
Jednak, gdy nie potrzebujesz alternatywnych to wystarczy:
```
Case.objects.filter(pk='5').all()
```

``cond`` był dotąd potrzebny, aby zbudować sobie obiekt ```Q```, aby potem wyszukiwać całość w jednym zapytaniu. Teraz można od razu wywołać metodę ``QuerySet``-u bez komplikowania z obiektem ```Q``.